### ISSUE TITLE
Configuration of publishing extension

### ISSUE THREAD

**Issue by Tallic** [bintray/gradle-bintray-plugin#239]
Hey there,
there is a regression with 1.8.1 that results in an invalid Gradle build.
`maven-publish` and `com.jfrog.bintray` 1.8.1 works fine, however, in conjunction with the `java-gradle-plugin` you will get the following error:

> Cannot configure the 'publishing' extension after it has been accessed.

MWE
```groovy
plugins {
    id "java"
    id "maven-publish"
    id "java-gradle-plugin"
    id "com.jfrog.bintray" version "1.8.1" // Note 1
}

publishing {
    publications {
        maven(MavenPublication) {
            from components.java

            groupId "some.group"
            artifactId "awesome"
            version "0.1.0"
        }
    }
}

bintray{
    publications = ["maven"] // Note 2
}
```
**Note 1:** using 1.8.0 works however then you cannot use Gradle 4.8 because of the compatibility issues in the bintray upload (extension uses lazy API).
**Note 2:** Commenting this will lead to a valid build.

**Environment:**

- Gradle 4.8
- com.jfrog.bintray 1.8.1

**Comment by eyalbe4** [comment]
Thanks for reporting this @Tallic.
We'll look into this soon.

**Comment by AlexeiVainshtein** [comment]
Hi @Tallic 

I just wanted to update you:
I've tried the same with Gradle 4.6 and plugin version 1.8.1 and it worked:

```Uploading to https://api.bintray.com/content/avjfrog/maven2/this_is_new2223/1.9-Final-Modules/org/jfrog/example/gradle/gradle-bintray-plugin-example/5.7.5/gradle-bintray-plugin-example-5.7.5.jar...
Uploaded to 'https://api.bintray.com/content/avjfrog/maven2/this_is_new2223/1.9-Final-Modules/org/jfrog/example/gradle/gradle-bintray-plugin-example/5.7.5/gradle-bintray-plugin-example-5.7.5.jar'.
Uploading to https://api.bintray.com/content/avjfrog/maven2/this_is_new2223/1.9-Final-Modules/org/jfrog/example/gradle/gradle-bintray-plugin-example/5.7.5/gradle-bintray-plugin-example-5.7.5.pom...
Uploaded to 'https://api.bintray.com/content/avjfrog/maven2/this_is_new2223/1.9-Final-Modules/org/jfrog/example/gradle/gradle-bintray-plugin-example/5.7.5/gradle-bintray-plugin-example-5.7.5.pom'.
:bintrayUpload (Thread[Task worker for ':',5,main]) completed. Took 3.765 secs.
:bintrayPublish (Thread[Task worker for ':',5,main]) started.
> Task :bintrayPublish 
Task ':bintrayPublish' is not up-to-date because:
  Task has not declared any outputs.
Published 'avjfrog/maven2/this_is_new2223/1.9-Final-Modules'.
```

However, with Gradle 4.8 indeed it fails:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'publications-example'.
> Cannot configure the 'publishing' extension after it has been accessed.
```
We are looking into this issue.

Thanks,

**Comment by AlexeiVainshtein** [comment]
Hi @Tallic

I just wanted to let you know that we have just released a new version 1.8.2 that should fix this issue. Can you please try this version and let us know if this fix's your issue?

Thanks,

**Comment by Tallic** [comment]
A nice thanks @AlexeiVainshtein for the fast fix. Everything is fine on my side.
